src/cloudai/cli/handlers.py

```diff
@@ -129,23 +185,34 @@ def handle_dry_run_and_run(args: argparse.Namespace) -> int:
 
     logging.info(test_scenario.pretty_print())
 
-    runner = Runner(args.mode, system, test_scenario)
-    asyncio.run(runner.run())
-
-    logging.info(f"All test scenario results stored at: {runner.runner.output_path}")
+    tr = next(iter(test_scenario.test_runs), None)
```

Comment:
We have pydantic check for it, test scenario can't contain zero cases, at least one is required. If it doesn't work, it should be fixed on model level.